sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/TimeSeriesInsightsClientSample/InstancesSamples.cs

```diff
@@ -0,0 +1,158 @@
+            var instanceIdsToGet = new List<TimeSeriesId>
+            {
+                instanceId,
+            };
+
+            Response<InstancesOperationResult[]> getInstancesByIdResult = await client.Instances.GetAsync(instanceIdsToGet).ConfigureAwait(false);
+
+            TimeSeriesInstance instanceResult = getInstancesByIdResult.Value[0].Instance;
+            Console.WriteLine($"Retrieved Time Series Insights instance with Id '{instanceResult.TimeSeriesId}' and name '{instanceResult.Name}'.");
+
+            // Now let's replace the instance with an updated name
+            instanceResult.Name = "newInstanceName";
+
+            var instancesToReplace = new List<TimeSeriesInstance>
+            {
+                instanceResult,
+            };
+
+            Response<InstancesOperationResult[]> replaceInstancesResult = await client.Instances.ReplaceAsync(instancesToReplace).ConfigureAwait(false);
+
+            // The response of calling the API contains a list of error objects corresponding by position to the input parameter
+            // array in the request. If the error object is set to null, this means the operation was a success.
+            if (replaceInstancesResult.Value[0].Error != null)
+            {
+                Console.WriteLine($"Failed to retrieve a Time Series Insights instnace with Id '{replaceInstancesResult.Value[0].Error.Message}'.");
+            }
+
+            #endregion Snippet:TimeSeriesInsightsReplaceInstance
+
+            #region Snippet:TimeSeriesInsightsGetnstancesById
+
+            // Get Time Series Insights instances by Id
+            var timeSeriesIds = new List<TimeSeriesId>
+            {
+                instanceId,
+            };
+
+            Response<InstancesOperationResult[]> getInstancesByNameResult = await client.Instances.GetAsync(timeSeriesIds).ConfigureAwait(false);
+
+            /// The response of calling the API contains a list of instance or error objects corresponding by position to the array in the request.
+            /// Instance object is set when operation is successful and error object is set when operation is unsuccessful.
+            InstancesOperationResult getInstanceByIdResult = getInstancesByNameResult.Value[0];
+            if (getInstanceByIdResult.Instance != null)
+            {
+                Console.WriteLine($"Retrieved Time Series Insights instance with Id '{getInstanceByIdResult.Instance.TimeSeriesId}' and name '{getInstanceByIdResult.Instance.Name}'.");
+            }
+            else if (getInstanceByIdResult.Error != null)
+            {
+                Console.WriteLine($"Failed to retrieve a Time Series Insights instnace with Id '{getInstanceByIdResult.Error.Message}'.");
+            }
+
+            #endregion Snippet:TimeSeriesInsightsGetnstancesById
+
+            // Clean up
+            try
+            {
+                #region Snippet:TimeSeriesInsightsSampleDeleteInstance
```

Comment:
I would rename snippet to DeleteInstanceById

Reply:
Also fine if want to keep it short, in which case change other snippet names as well.